app/components/applications-app-services/variables.tf

```diff
@@ -39,11 +39,6 @@ variable "applications_sql_server_connection_string" {
   type        = string
 }
 
-variable "applications_sql_server_connection_string_jbdc" {
-  description = "Connection string for applications service sql server in JBDC format"
-  type        = string
-}
-
```

Comment:
I think you can also remove `variable "applications_sql_server_connection_string"`. I can't see it being used anywhere within `app/components/applications-app-services`

Reply:
TFLint is picking this up actually:

`TFLint in app/components/applications-app-services/:
1 issue(s) found:

Warning: [Fixable] variable "applications_sql_server_connection_string" is declared but not used (terraform_unused_declarations)

  on variables.tf line 37:
  37: variable "applications_sql_server_connection_string" {

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.4.0/docs/rules/terraform_unused_declarations.md
`